src/WebJobs.Script.WebHost/App_Start/WebApiConfig.cs

```diff
@@ -114,10 +114,11 @@ private static WebHostSettings GetDefaultSettings()
             WebHostSettings settings = new WebHostSettings();
 
             string home = Environment.GetEnvironmentVariable("HOME");
+            string scriptPath = Environment.GetEnvironmentVariable("SCRIPTPATH");
```

Comment:
We added this environment variable to be able to point the functions root directory to a different folder on the local machine without changing the code. for our testing purposes we figured we were always changing this line hence added this environment variable.